tests/installation/check_releasenotes.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check releasenotes during installation
+# Maintainer: mgriessmeier <mgriessmeier@suse.com>, Nick Singer <nsinger@suse.de>
+# Tags: fate#323273, poo#26786
+
+use strict;
+use base "y2logsstep";
+use testapi;
+
+sub run {
+    assert_screen('release-notes-button');
+    send_key('ctrl-shift-alt-x');
+    assert_screen('yast-xterm');
+    my $src = check_var('SCC_REGISTER', 'installation') ? "RPM" : "URL";
+    type_string "grep -o \"Got release notes.*$src\" /var/log/YaST2/y2log\n";
+    assert_screen "got-releasenotes-$src";
+    type_string "exit\n";
+    assert_screen 'system-role-default-system', 180;
```

Comment:
hm, that is depending on the order in sle/main.pm where the test module is scheduled. Therefore I consider this not so nice. Maybe "installation" would be a better tag to check for. This is the generic one we use when switching back from a install-shell to to "installation" screen.

Reply:
Well, the fate clearly states that this should be done on system-role screen.
Though I understand your intention and agree in general

Reply:
I see the fate as a hint and mainly aimed at manual testing. What we do for automation can be reconsidered. Whoever wrote the testcase in fate probably does not know that we prefer tty consoles, e.g. to allow the s390x backend to connect over ssh and look the same using xterm-console :)